README.md

```diff
@@ -81,58 +81,68 @@ These final equations are the ones used in the function **compute_bz**, which ca
 
 ## Installation 
 
-First, clone the repository
+- Clone the repository
 
 ```
 git clone https://github.com/shimming-toolbox/susceptibility-to-fieldmap-fft.git
+cd susceptibility-to-fieldmap-fft
 ```
 
-Navigate to the project directory
+- Create a virtual environnement 
 
 ```
-cd susceptibility-to-fieldmap-fft
+conda create --name <name of your environement> python=3.9
+conda activate <name of your environment>
+```
+
+- Go to the develop branch (temporary)
+
+```
+git checkout develop
 ```
 
-Install the requirements
+- Install the package
 
 ```
-pip install -r requirements.txt
+pip install .
 ```
 
+You will need to ```conda activate <name of your environment>``` each time you want to use the package.
+
 ## Usage
-To execute the scripts, you need to naviagte into the fft_simulation folder.
+Once the package is installed, the commands can be run directly from the terminal. Here is the description of the two commands available.
 
-### Analytical cases
+### analytical_cases
 
-The _analytical_case.py_ script allows for comparaison between simulated and analytical results for a spherical and cylindrical phantom. 
+The _analytical_cases_ command allows for comparaison between simulated and analytical results for a spherical and cylindrical phantom. 
```

Comment:
```suggestion
The `analytical_cases` command allows for comparaison between simulated and analytical results for a spherical and cylindrical phantom. 
```

Reply:
it's not very clear to me what the function actually does. Can you give an example of figure output? Also, what should be my input here? it seems that -t and -b are not image inputs, so I'm puzzled as to what I'm supposed to input...?

Reply:
right, i'll add an example of figures output. the inputs -t and -b are defined in the **options** as the geometry type and the buffer value respectively.